ci_framework/playbooks/ceph.yml

```diff
@@ -180,6 +198,16 @@
         name: cifmw_cephadm
         tasks_from: pools
 
+    - name: Deploy RGW
+      ansible.builtin.import_role:
+        name: cifmw_cephadm
+        tasks_from: rgw
+      vars:
+        # public network always exist because is provided by the ceph_spec role
+        cifmw_cephadm_rgw_network: "{{ lookup('ansible.builtin.ini', 'public_network', section='global',  file=cifmw_cephadm_bootstrap_conf) }}"
+        # cifmw_cephadm_vip is computed or passed as an override via -e @extra.yml
+        cifmw_cephadm_rgw_vip: "{{ cifmw_cephadm_vip }}/24"
```

Comment:
@fultonj here we propagate the VIP: note that I'm using these variables because I can always run the `ceph.yml` playbook passing `-e @extra.yml` and override this data as needed.